test/providers/curation/github.js

```diff
@@ -51,6 +51,11 @@ describe('Github Curation Service', () => {
       return [createInvalidCuration()]
     })
     const curations = await service.getContributedCurations(42, 'testBranch')
+    service.logger = { // intercept and verify invalid contribution
+      error: function(description /* , invalidCurations */) {
```

Comment:
Let's change it to an arrow function `error: (description) =>`

Reply:
done